Initial install and setup - Cannot find module '../../package.json'

<!-- Requirements: please go through this checklist before opening a new issue -->

- [x] Review the documentation: https://docs.sentry.io/platforms/javascript/electron/
- [x] Search for existing issues: https://github.com/getsentry/sentry-electron/issues
- [x] Use the latest release: https://github.com/getsentry/sentry-electron/releases
- [x] Provide a link to the affected event from your Sentry account

## Versions + Platform

- [x] SDK version - `@sentry/electron@v2.5.3`
- [x] Electron version - `electron@v14.0.0`
- [x] Platform - `Linux`

## Description

Followed the initial setup from this page:

https://docs.sentry.io/platforms/javascript/guides/electron/

Added the following to the top of my main and preload scripts:

```js
import * as Sentry from "@sentry/electron";

Sentry.init({dsn: 'https://MY_DSN'});
```
I get this error:

```
A JavaScript error occurred in the main process
Uncaught Exception:
Error: Cannot find module '../../package.json'
Require stack:
- /home/davidwinter/projects/nimblenote/packages/main/dist/index.cjs
- /home/davidwinter/projects/nimblenote/node_modules/electron/dist/resources/default_app.asar/main.js
- 
    at Module._resolveFilename (internal/modules/cjs/loader.js:892:15)
    at Function.n._resolveFilename (electron/js2c/browser_init.js:245:1128)
    at Module._load (internal/modules/cjs/loader.js:737:27)
    at Function.f._load (electron/js2c/asar_bundle.js:5:12913)
    at Module.require (internal/modules/cjs/loader.js:964:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/home/davidwinter/projects/nimblenote/packages/main/dist/index.cjs:6994:19)
    at Module._compile (internal/modules/cjs/loader.js:1083:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1113:10)
    at Module.load (internal/modules/cjs/loader.js:940:32)
```

I assume this might be due to this line:

https://github.com/getsentry/sentry-electron/blob/edd2775048d4658f88fd23a989574da913e6762a/src/main/context.ts#L13

I'm using Vite with Electron.

I'm guessing the resolving of the `package.json` file for this module is failing. I've had this on one of my projects, and I followed guidance here: https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c#what-do-i-use-instead-of-__dirname-and-__filename

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,13 +2,17 @@
 
 ## Unreleased
 
+## 2.5.4
+
+- fix: Improve compatibility with bundlers
+
 ## 2.5.3
 
--fix: Possible race condition with `initialScope` over IPC
+- fix: Possible race condition with `initialScope` over IPC
 
 ## 2.5.2
 
--fix: Add `release` and `environment` to electron uploader `initialScope`
+- fix: Add `release` and `environment` to electron uploader `initialScope`
 
 ## 2.5.1
 

--- a/package.json
+++ b/package.json
@@ -16,6 +16,7 @@
     "access": "public"
   },
   "scripts": {
+    "prebuild": "node scripts/update-version.js",
     "build": "run-p build:es6 build:esm",
     "build:es6": "tsc -p tsconfig.build.json",
     "build:esm": "tsc -p tsconfig.esm.json",

--- a/scripts/update-version.js
+++ b/scripts/update-version.js
@@ -1,0 +1,8 @@
+const { join } = require('path');
+const { readFileSync, writeFileSync } = require('fs');
+
+const version = require('../package.json').version;
+const path = join(__dirname, '../src/main/version.ts');
+const code = readFileSync(path, { encoding: 'utf8' });
+const modified = code.replace(/SDK_VERSION = '[\d\.]*'/, `SDK_VERSION = '${version}'`);
+writeFileSync(path, modified);

--- a/src/main/context.ts
+++ b/src/main/context.ts
@@ -7,10 +7,7 @@ import { join } from 'path';
 
 import { getNameFallback } from '../common';
 import { readDirAsync, readFileAsync } from './fs';
-
-/** SDK version used in every event. */
-// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-export const SDK_VERSION: string = require('../../package.json').version;
+import { SDK_VERSION } from './version';
 
 export const SDK_NAME = 'sentry.javascript.electron';
 

--- a/src/main/version.ts
+++ b/src/main/version.ts
@@ -1,0 +1,1 @@
+export const SDK_VERSION = '2.5.3';

